services/meta/data_test.go

```diff
@@ -215,6 +215,104 @@ func TestData_SetPrivilege(t *testing.T) {
 	}
 }
 
+func TestData_TruncateShardGroups(t *testing.T) {
+	data := &meta.Data{}
+
+	must := func(err error) {
```

Comment:
I think I'd rather see this as `assertNil` or something. `must` usually implies that if there is an error you `panic`.

Reply:
Another approach we often take in the database (though it seems overkill for one test) is to create a `Data` type in the test, and then write methods like `MustCreateDatabase`, `MustCreateShardGroup` etc.